### ISSUE TITLE
Alternately-encoded file detected as binary

### ISSUE THREAD

**Issue by miketheman** [pypi/inspector#187]
https://inspector.pypi.io/project/algfastcoding/31/packages/38/9f/44b9b5c8c32019f682d3c21c3260bec8111306cbaac171aea66c7300a2e7/algfastcoding-31.tar.gz/algfastcoding-31/setup.py is detected as binary, and will not display.

Upon deeper inspection, the file is detected as:

```shellsession
$ file algfastcoding-31/setup.py
algfastcoding-31/setup.py: Python script text executable, ISO-8859 text, with CRLF line terminators
$ head algfastcoding-31/setup.py
# -*-coding:GBK-*-
"""
* ���ߣ�������
* ʱ�䣺2022/1/25 14:00
* ���ܣ����Python��������ڷ�����pypi.org
* ˵���뿴����.txt���ⷢ�����ʹ��ѧ��˼�������������
"""
import sys

from setuptools import setup
```

Inspector should handle other encodings if possible

**Comment by miketheman** [comment]
Hit this again today, with a detected magic of: 
Python script, ISO-8859 text executable, with CRLF line terminators
